go.mod

```diff
@@ -16,6 +16,8 @@ require (
 	github.com/gorilla/mux v1.8.0
 	github.com/iancoleman/strcase v0.1.3
 	github.com/imdario/mergo v0.3.12
+	github.com/instrumenta/kubeval v0.16.1
+	github.com/jonboulle/clockwork v0.1.0
```

Comment:
What is this dep used for? It is not marked as indirect and I do not see any direct usage of the package in the PR 🤔